aml-book/contents/lecture1-introduction.ipynb

```diff
@@ -148,7 +148,7 @@
     "\n",
     "> Machine learning is a field of study that gives computers the ability to learn without being explicitly programmed.\n",
     "\n",
-    "This definition is used in numerous courses and textbooks. At first, it might be hard to grasp what Samuel means by terms like \"learn\" and \"explicitly programmed\". Let's look at one final example to clarify this."
+    "This definition is used in numerous courses and textbooks. At first, it might be hard to grasp what Samuel means by terms like \"learn\" and \"explicitly programmed.\" Let's look at one final example to clarify this."
```

Comment:
Having the punctuation after double inverted comma ending is more appropriate. Could you revert PR changes corresponding to this typo throughout the documents?

Reply:
In the lecture notes, I found some periods and commas inside closing quotation marks and others outside. I switched the ones that were outside to bring them inside since in grammatically, in Standard American English, commas and periods are placed inside the closing quotation mark. Regardless of whether the commas and periods are inside or outside the closing quotation marks though, this should be consistent across all of the lecture notes. 

[https://owl.purdue.edu/owl/general_writing/punctuation/punctuation/index.html#:~:text=Note%20that%20commas%20and%20periods,and%20semicolons%20are%20placed%20outside](https://owl.purdue.edu/owl/general_writing/punctuation/punctuation/index.html#:~:text=Note%20that%20commas%20and%20periods,and%20semicolons%20are%20placed%20outside)

Reply:
I am not sure what happened with lecture 12, but I will try to fix it. Can you please approve the other changes meanwhile?

Reply:
Thank you for clarifying the suggestions on quotations. Since I am used to British English, I did not realise this difference in American English. 

Please do add a comment here when you modify lecture 12. At the moment, the diff is so huge that it is impossible to see what changed. 

I will approve the other changes.